starters/express-typescript-typeorm-postgres/src/utils/log-helper.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const allowedLogLevels = process.env.ALLOWED_LOG_LEVELS || '[]';
+
+type LogLevel = 'INFO' | 'DEBUG' | 'WARN' | 'ERROR';
+
+let LOG_LEVELS: LogLevel[] = ['DEBUG', 'INFO', 'WARN', 'ERROR'];
+try {
+  const parsedLogLevels = JSON.parse(allowedLogLevels);
```

Comment:
We probably don't need to parse this into an object, we could probably just check `process.env.ALLOWED_LOG_LEVELS` for the strings, then people could do comma separated or whatever
```ts
new RegExp(LOG_LEVEL).test(process.env.ALLOWED_LOG_LEVELS)
```

Reply:
I recently brought down a production environment with using regexes in logging. I'd feel safer with going `.split(',')` and putting that into a Set.